src/app/shared/components/filters-list/searchbar/searchbar.component.ts

```diff
@@ -47,14 +46,14 @@ export class SearchbarComponent implements OnInit, OnDestroy {
         map((value: string) => value.trim()),
         tap((value: string) => this.filter(value))
       )
-      .subscribe((value: string) => (this.searchedText = value));
+      .subscribe((value: string) => (this.searchedText = value.trim()));
 
     this.searchQuery$
       .pipe(takeUntil(this.destroy$))
       .subscribe((text: string) => this.searchValueFormControl.setValue(text, { emitEvent: false }));
 
     if (!this.isResultPage) {
-      this.searchValueFormControl.setValue('', { emitEvent: false });
+      this.resetSearchInput();
```

Comment:
Why do we extract one line function call to a separate method, I can't find any other usages of resetSearchInput?
It would be more useful to add some explanation comment